app/src/forms/rbac/routes.js

```diff
@@ -9,42 +9,578 @@ const { currentUser, hasFormPermissions, hasSubmissionPermissions, hasFormRoles,
+ *        required: true
+ *        example: c6455376-382c-439d-a811-0381a012d696
+ *      - in: path
+ *        name: userId
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *        description: User ID
+ *        required: true
+ *        example: c6455376-382c-439d-a811-0381a012d696
+ *    requestBody:
+ *      required: true
+ *      description: OK
+ *      content:
+ *        application/json:
+ *          schema:
+ *            type: array
+ *            items:
+ *              type: object
+ *              properties:
+ *                formId:
+ *                  type: string
+ *                  description: Form ID
+ *                  format: uuid
+ *                  example: e37705ee-1c01-44eb-ae8a-e7d96002ae67
+ *                role:
+ *                  type: string
+ *                  description: Assigned role to user on the form
+ *                  example: owner
+ *                userId:
+ *                  type: string
+ *                  description: ID of the User
+ *                  format: uuid
+ *                  example: 2a4770b7-6f0e-4629-a359-bf820295f23a
+ *    responses:
+ *      '200':
+ *        description: Success
+ *        content:
+ *          application/json:
+ *            schema:
+ *              $ref: '#/components/responses/responseBody/RBACSetUsersForms'
+ *      '401':
+ *        description: Unauthorized
+ *        content:
+ *          application/json:
+ *            schema:
+ *              oneOf:
+ *                - $ref: '#/components/schemas/respError/NoFormAccessError'
+ *                - $ref: '#/components/schemas/respError/UserNotFoundError'
+ *                - $ref: '#/components/schemas/respError/FormIdNotFoundError'
+ *                - $ref: '#/components/schemas/respError/NoRequiredFormPermissionError'
+ *      '403':
+ *        $ref: '#/components/responses/Error/AccessDenied'
+ *      '5XX':
+ *        $ref: '#/components/responses/Error/UnExpected'
+ */
 routes.put('/users', hasFormPermissions(P.TEAM_UPDATE), hasFormRoles([R.OWNER, R.TEAM_MANAGER]), hasRolePermissions(false), async (req, res, next) => {
   await controller.setUserForms(req, res, next);
 });
 
+/**
+ * @openapi
+ * rbac/users:
+ *  delete:
+ *    tags:
+ *      - RBAC
+ *    description: This endpoint will remove multiple users from a form.
+ *    security:
+ *      - bearerAuth: []
+ *      - openId: []
+ *    parameters:
+ *      - in: path
+ *        name: formId
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *        description: Form ID.
+ *        required: true
+ *        example: c6455376-382c-439d-a811-0381a012d696
+ *    requestBody:
+ *      required: true
+ *      content:
+ *        application/json:
+ *          schema:
+ *            type: array
+ *            example: ["7cc11a0f-a7c9-4cbe-bd92-a3a22781ac8c", "3ddb68cb-b936-49fe-9ba4-48b04859eaf7" ]
+ *    responses:
+ *      '200':
+ *        description: Sucess
```

Comment:
Typo here: "Sucess" instead of "Success".

Reply:
Fixed